packages/flutter/lib/src/widgets/size_changed_layout_notifier.dart

```diff
@@ -73,6 +85,8 @@ class _RenderSizeChangedWithCallback extends RenderProxyBox {
   @override
   void performLayout() {
     super.performLayout();
+    // Don't send the initial notification, or this will be SizeObserver all
+    // over again!
```

Comment:
Do you want to link to the bug were we decided to remove size observer?  Folks who read this comment in the future are unlikely to know what size observer was without more context.